cmd/minikube/cmd/start.go

```diff
@@ -519,46 +520,58 @@ func selectDriver(existing *config.ClusterConfig) registry.DriverState {
 	return pick
 }
 
+// hostDriver returns the true driver used without relying on config fields
```

Comment:
the comment is not clear to me, what is a True driver, 
maybe elaborate more on that.

Reply:
it just means he's grabbing the driver from the machine itself rather than the config

Reply:
I rather comments that everyone can undrestand, including new contributors